From 0000000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: hansendc <hansendc@example.com>
Date: Mon, 26 Feb 2026 17:21:46 +0000
Subject: [PATCH 1/3] add fullscreen state

Added a fullscreen state variable to manage fullscreen mode in the viewer.

--- 
 viewer.c | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/viewer.c
+++ b/viewer.c
@@ -1,5 +1,11 @@
 #include <stdio.h>
 
 // Example viewer code
+
+int fullscreen_state = 0; // 0 = normal, 1 = fullscreen
+
 void render() {
     // rendering function
 }  
